[2024-06-11T05:39:02.535+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T05:38:53.028335+00:00 [queued]>
[2024-06-11T05:39:02.539+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T05:38:53.028335+00:00 [queued]>
[2024-06-11T05:39:02.539+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T05:39:02.540+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-11T05:39:02.540+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T05:39:02.548+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task4> on 2024-06-11 05:38:53.028335+00:00
[2024-06-11T05:39:02.551+0000] {standard_task_runner.py:55} INFO - Started process 1051 to run task
[2024-06-11T05:39:02.553+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task4', 'manual__2024-06-11T05:38:53.028335+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmpay7c0jmt']
[2024-06-11T05:39:02.553+0000] {standard_task_runner.py:83} INFO - Job 173: Subtask task4
[2024-06-11T05:39:02.583+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T05:38:53.028335+00:00 [running]> on host 90158f1b5502
[2024-06-11T05:39:02.615+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task4
AIRFLOW_CTX_EXECUTION_DATE=2024-06-11T05:38:53.028335+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-11T05:38:53.028335+00:00
[2024-06-11T05:39:02.615+0000] {logging_mixin.py:120} INFO - Task 4 is completed
[2024-06-11T05:39:04.619+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-11T05:39:04.644+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task4, execution_date=20240611T053853, start_date=20240611T053902, end_date=20240611T053904
[2024-06-11T05:39:04.686+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-11T05:39:04.704+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
